squire_lib/src/tournament.rs

```diff
@@ -971,19 +962,26 @@ impl Tournament {
 
 impl TournamentSeed {
     /// Creates a new tournament seed
-    pub fn new(name: String, preset: TournamentPreset, format: String) -> Self {
-        Self {
+    pub fn new(
+        name: String,
+        preset: TournamentPreset,
+        format: String,
+    ) -> Result<Self, TournamentError> {
+        // name validation
+        if name.is_empty() || !name.is_ascii() {
```

Comment:
Restricting names to ascii seems like a harsh restriction. Is there a particular reason for this? With the exception of the desktop, tournament names rarely leave Rust, where we can safely handle non-ascii strings.

Reply:
No particular reason, if it's not needed then I'll remove it.